app/.eslintrc.js

```diff
@@ -15,5 +15,10 @@ module.exports = exports = {
     rules: {
         'react-hooks/rules-of-hooks': 'error',
         'react-hooks/exhaustive-deps': 'warn',
+        'no-unused-vars': 'warn',
+        'no-multiple-empty-lines': 'warn',
+        'jsx-a11y/no-noninteractive-tabindex': 'warn',
+        'react/jsx-indent': 'warn',
+        'max-len': 'warn',
```

Comment:
Curious why you needed to change these.
Maybe should discuss or ideally disable on a case-by-case basis.
i.e. If they're just warnings maybe we just should turn them off otherwise the warnings are meaningless noise?

Reply:
I want more rules to guide, less to enforce – gives more freedom in prototyping.

And noise isn't entirely meaningless. I do care that my builds are green. Warnings bother me. But when I'm playing around and my line happens to be too long, and it blocks me…, man, that's annoying.

I, btw, don't wanna use `/* eslint-disable */`. It tends to be overused or forgotten, or applied too widely.

Hope it makes sense. :)

Reply:
`error` -> `warn` for these are a great idea IMO 👍 I too like to live dangerously.

Reply:
I do hope that builds keep somewhat clean of warnings though, I suppose the negative here is that there could be an accumulation of warnings from various merges. Still in favour!